debian/changelog

```diff
@@ -1,3 +1,9 @@
+system76-firmware (1.0.44) jammy; urgency=medium
```

Comment:
Should this be kept as `focal`?

Reply:
Oh, probably. `dch` changed it on me, I'll change it back real quick